.github/workflows/install.yml

```
@@ -531,8 +531,78 @@ jobs:
           
           # 清理缓存
           "$PY_EXEC" -m pip cache purge
-          find install/python -name "__pycache__" -type d -exec rm -rf {} +     
+          find install/python -name "__pycache__" -type d -exec rm -rf {} +
 
+          # 🔧 macOS 便携 Python 的 pip 安装时可能丢失 wheel 内的 .dylibs 目录
+          # (如 numpy/.dylibs/libopenblas64_.0.dylib, PIL/.dylibs/libtiff.6.dylib 等)
+          # 此处扫描所有 .so 文件的 @loader_path 依赖,缺失则从对应 wheel 中提取补回
+          echo "🔍 Verifying native dylibs..."
+          "$PY_EXEC" -c "
+import os, sys, subprocess, tempfile, zipfile, shutil, re, importlib.metadata
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)